Tribler/Core/Modules/restapi/rest_manager.py

```diff
@@ -30,7 +30,7 @@ def start(self):
         self.root_endpoint = RootEndpoint(self.session)
         site = server.Site(resource=self.root_endpoint)
         site.requestFactory = RESTRequest
-        self.site = reactor.listenTCP(self.session.get_http_api_port(), site)
+        self.site = reactor.listenTCP(self.session.get_http_api_port(), site, interface="127.0.0.1")
```

Comment:
We could make it a configuration option if we need more flexibility (which we might need in the end since controlling/monitoring a remote Tribler instances is quite powerful). What about making it a config option which defaults to localhost? @egbertbouman do you agree with that?